### ISSUE TITLE
Issue status transition to apply when a time tracker is stoped

### ISSUE THREAD

**Issue by kirill-scherba** [fernandokosh/redmine_time_tracker#69]
It's very useful to add option "Issue status transition to apply when a time tracker is stoped" to config of the PlugIn

**Comment by delaitre** [comment]
I don't think this is really useful/needed. When you stop the time tracker, you already are able to change the status of the issue, along with the percentage done... However usually you don't want (well, at least I don't) to change the status automatically as you can work on the same issue several times.

If I implement this option this will be this way:
- user can choose to select a "default stop status" when stopping a time tracker OR to leave the current status
- when stopping the time tracker and a "default status status" is selected, select this status from the combo box in the time logging page. This way, the status can still be adjusted
- when stopping the time tracker and the "leave current status" option is chosen, keep the current behavior (select the current status in the combo box)

**Comment by kirill-scherba** [comment]
The change of start status ia optional & the change of stop status may be optional too. When I stop work with my Issue, I switch it to feedback status most of all, because I can't execute Issue at one time. So change stop status will be very useful for me. Stanks.
